frontend/app/scripts/views/layouts/home_layout_view.js

```diff
@@ -85,9 +103,10 @@ define([
     },
 
     renderErrorView: function() {
```

Comment:
Modify view to be a "show message" utility view, instead of a show `error` message view. Anyone who would like to show a particular message should be able to append that view to the DOM.